README.md

```diff
@@ -1,58 +1,14 @@
-[![CircleCI](https://circleci.com/gh/facebookincubator/Polygames.svg?style=svg)](https://circleci.com/gh/facebookincubator/Polygames)
-
 # Polygames
 
-This README is a work in progress, please feel very free to post issues - we are happy to help.
-Save up computational power: you can find checkpoints here: http://dl.fbaipublicfiles.com/polygames/checkpoints/list.txt (feel free to open an issue for discussing which checkpoint you should use for which game/problem!).
-
-For Nix users: see [this doc](./nix/README.md).
-
-## Requirement:
-```
-C++17 compatible compiler
-miniconda3
-```
-
-## Compilation Guide:
-
-### First install conda and pytorch
-
-Create a fresh conda environment with python3.7, install pytorch and dependencies.
-
-```
-# create a fresh conda environment with python3
-# you will need to have miniconda3 set up
-conda create --name [your env name] python=3.7 pip
-
-conda activate [your env name] # Or source activate [your env name], depending on conda version.
-
-conda install numpy pyyaml mkl mkl-include setuptools cmake cffi typing
-conda install pytorch cudatoolkit=10.1 -c pytorch
-conda install -c conda-forge tensorboardx
-conda install -c conda-forge openjdk  # optional
-conda install -c conda-forge graphviz # optional
-
-pip install visdom
-pip install torchviz				  # optional
-
-```
-
-### Clone the repo and build
-
+This repository is a fork of https://github.com/facebookarchive/Polygames.
 
-```
-git clone --recursive https://github.com/facebookincubator/polygames
-cd polygames
+## Installation
 
-mkdir build
-cd build
+To get the code up and running, see the dockerfile and instructions in https://github.com/AlignmentResearch/polygames-scripts.
 
-cmake .. -DCMAKE_BUILD_TYPE=relwithdebinfo -DPYTORCH15=ON
-make -j
-
-```
+## Checkpoints
 
-Ludii support can be disabled by appending `-DWITH_LUDII=OFF` to the cmake command (required if you don't have jdk)
+http://dl.fbaipublicfiles.com/polygames/checkpoints/list.txt.
```

Comment:
might want to elaborate on what this is, it's not immediately clear without context